src/ontology/mondo-edit.obo

```diff
@@ -263531,23 +263531,27 @@ property_value: confidence "2.0694736842105272" xsd:double
+name: CBL-related disorder
+def: "CBL-related disorder is a genetic condition caused by pathogenic variants in the Cbl ubiquitin ligase gene, (CBL; HGNC:1541). Due to the proposed mechanism indicating the CBL gene's relationship to the RAS-MAPK pathway and the phenotypic presentation similar to that of the RASopathies, CBL-related disorder should be considered a RASopathy disorder. Though there is a wide spectrum of phenotypic variability, broadly, patients with CBL-related disorder have presented with developmental delay, intellectual disability, neurodevelopmental alterations, prenatal lymphatic anomalies, cardiac malformations as well as vascular anomalies particularly affecting the brain (e.g. Moya-moya arteriopathies), craniofacial features indicative of a RASopathy, hypotonia, feeding difficulties, edema of the legs, musculoskeletal and respiratory thorax abnormalities, ectodermal features including cafe-au-lait spots, immunological and hematological disorders and susceptibility to tumors diagnosed as juvenile myelomonocytic leukemia (JMML) that is usually self-remitting. Note tumor risk beyond JMML has not yet been thoroughly assessed. Due to the clinical presentation of a broad spectrum of these and other phenotypes in patients with variants in CBL, these conditions are currently defined by experts in reference to the causal gene, CBL." [PMID:20619386, PMID:21901340, PMID:24458550, PMID:25283271, PMID:25358541, PMID:25952305, PMID:26911351, PMID:28343148, PMID:28414188, PMID:28589114, PMID:29259247]
 subset: ordo_malformation_syndrome {source="Orphanet:363972"}
 synonym: "Cbl mutation-associated syndrome" RELATED [OMIM:613563]
 synonym: "Cbl syndrome" EXACT [OMIM:613563, Orphanet:363972]
 synonym: "Noonan syndrome-like disorder with JMML" EXACT [Orphanet:363972]
+synonym: "Noonan syndrome-like disorder with juvenile myelomonocytic leukemia" EXACT []
 synonym: "Noonan syndrome-like disorder with or without juvenile myelomonocytic leukemia" RELATED [MONDO:Lexical, OMIM:613563]
 synonym: "Noonan syndrome-like disorder with or without juvenile myelomonocytic leukemia; NSLL" RELATED [OMIM:613563]
+synonym: "Noonan-like syndrome" EXACT []
 synonym: "NSLL" RELATED [MONDO:Lexical, OMIM:613563]
 xref: ICD10:Q87.1 {source="ORDO:363972/attributed", source="ORDO:363972/ntbt", source="Orphanet:363972"}
 xref: OMIM:613563 {source="ORDO:363972/e", source="MONDO:equivalentTo", source="Orphanet:363972"}
 xref: Orphanet:363972 {source="OMIM:613563", source="MONDO:equivalentTo"}
 xref: UMLS:C3150803 {source="OMIM:613563", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
 xref: UMLS:C4016301 {source="MEDGEN:kboom-pr98-c99", source="MONDO:equivalentTo"}
 is_a: MONDO:0015945 {source="Orphanet:363972"} ! polymalformative genetic syndrome with increased risk of developing cancer
-is_a: MONDO:0020297 {source="Orphanet:363972"} ! Noonan syndrome and Noonan-related syndrome
+is_a: MONDO:0021060 {source="ClinGen RASopathy Expert Panel"} ! RASopathy
```

Comment:
Source should be a CURIE or URI

Reply:
@cmungall what should I use for the source in this case?  There are 14 ORCIDs listed on the ticket for the panel members, should I add all of those ORCIDs?

Reply:
No, not 14!

Is there a URL for the group? If not I suggest setting up one.

But for now, use a GO-style meeting CURIE e.g. ClinGen:RASopathyExpertPanel. No spaces. Let's also collect a doc with full metadata on this organization (but that can be done outside this PR)

Reply:
Thanks - I updated the source. There is not a URL that I know of.
Here is a google doc with the names/ORCIDs for this group: https://docs.google.com/document/d/1t4cj1MZ9xlONOqsScQwbniTprMj83C4Zur1T_na9HMM/edit

Reply:
We can host a URL on mondo site if CLinGen can't

Reply:
Ok, I'll ask ClinGen